Jenkinsfile

```diff
@@ -11,14 +11,31 @@ pipeline {
         dockerfile true
     }
 
+    parameters {
+        string(name: "REMOTE_SWF_LINK", defaultValue: "https://github.com/Pelagicore/software-factory.git")
+        string(name: "REMOTE_SWF_BP_LINK", defaultValue: "https://github.com/Pelagicore/software-factory-blueprint.git")
+        string(name: "BRANCH_SWF", defaultValue: "master")
+    }
+
     stages {
         stage('Download') {
             steps {
+
                 // Delete old files
+
                 sh 'rm -rf .[^.] .??* *'
 
-                // Checkout the git repository and refspec pointed to by jenkins
-                checkout scm
```

Comment:
Would it be possible to make these steps conditional on there being a CHANGE_ID?

Imho, it makes sense to do the regular build in regular cases, and the build that this PR introduces only when there is a SWF BP change done (so pull requests, then).

Or does this work properly when building static branches as well, such as master? I'm not sure.